labs/blockchain-explorer.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Blockchain Explorer
+parent: Labs
+---
+# Lab Name
+Blockchain Explorer
+
+# Short Description
+Blockchain Explorer is a simple, powerful, easy-to-use, well-maintained, open-source utility to browse activity 
+on the underlying blockchain network. 
+
+# Scope of Lab
+- Enhance the existing application in-terms of new features
+- Make it compatible with latest version of Hyperledger Fabric 
+- Uplift the user-interface with the robust design
+- Test-cases and documentation
+
+# Initial Committers
+- https://github.com/ryjones
```

Comment:
@ArchanaArige please add the current set of maintainers that we want to start the labs with. These are previous committers, their contributions are counted but they would not be a maintainer as of now.

Reply:
Hi @arsulegai. Thanks for the review. As per the proposal-template.md there are only five sections where we don't have #current maintainers. Please confirm whether we can add a separate section as #current maintainers to the proposal as we suggested.

Reply:
@ArchanaArige as per the discussion outcome in discord. @tkuhrt too agrees that we can replace this list by adding current set of maintainers.

Reply:
@arsulegai , I have added the current-maintainers section, in-addition to the initial-committers.
Although Hyperledger-Explorer coined ad EOL, it's good to have the initial-committers section who have contributed to this project initially. If there has to be any changes, please do let me know.

Reply:
Adding AdityaJoshi to the current maintainers list.